src/status_im/constants.cljs

```diff
@@ -89,24 +86,12 @@
           mainnet-networks)))
 
 (def default-wnodes
-  {:testnet {"main" {:id      "main"
-                     :name    "Status mailserver A"
-                     :address "enode://1da276e34126e93babf24ec88aac1a7602b4cbb2e11b0961d0ab5e989ca9c261aa7f7c1c85f15550a5f1e5a5ca2305b53b9280cf5894d5ecf7d257b173136d40@167.99.209.61:30504"}
-             "backup" {:id      "backup"
-                       :name    "Status mailserver B"
-                       :address "enode://07ac64fe0e9b2d4ecbfe0ccaeab3d8d95fa8858754f511104bb403b19255d7bf47a8416bdd0df01f6720ff82164451b7a028c93d15ddd37b7e8382d74e91ebc2@167.99.209.72:30504"}}
-   :mainnet {"main" {:id      "main"
-                     :name    "Status mailserver A"
-                     :address "enode://1da276e34126e93babf24ec88aac1a7602b4cbb2e11b0961d0ab5e989ca9c261aa7f7c1c85f15550a5f1e5a5ca2305b53b9280cf5894d5ecf7d257b173136d40@167.99.209.61:30504"}
-             "backup" {:id      "backup"
-                       :name    "Status mailserver B"
-                       :address "enode://07ac64fe0e9b2d4ecbfe0ccaeab3d8d95fa8858754f511104bb403b19255d7bf47a8416bdd0df01f6720ff82164451b7a028c93d15ddd37b7e8382d74e91ebc2@167.99.209.72:30504"}}
-   :rinkeby {"main" {:id     "main"
-                     :name   "Status mailserver A"
-                     :address "enode://43829580446ad138386dadb7fa50b6bd4d99f7c28659a0bc08115f8c0380005922a340962496f6af756a42b94a1522baa38a694fa27de59c3a73d4e08d5dbb31@206.189.6.48:30504"}
-             "backup" {:id      "backup"
-                       :name    "Status mailserver B"
-                       :address "enode://70a2004e78399075f566033c42e9a0b1d43c683d4742755bb5457d03191be66a1b48c2b4fb259696839f28646a5828a1958b900860e27897f984ad0fc8482404@206.189.56.154:30504"}}})
+  {"main" {:id      "main"
+           :name    "Status mailserver A"
+           :address "enode://1da276e34126e93babf24ec88aac1a7602b4cbb2e11b0961d0ab5e989ca9c261aa7f7c1c85f15550a5f1e5a5ca2305b53b9280cf5894d5ecf7d257b173136d40@167.99.209.61:30504"}
+   "backup" {:id      "backup"
```

Comment:
At this point it does not seem to be a backup anymore, as I understand from the code it randomly choose one on create/restore account ( I understand it has a preference for `main` for existing accounts), but just by the name I would expect something called `backup` to be used when the primary fails or in special circumstances.

I would change the name to A/B, so it gives a clearer message, and we can easily add C if needed at some point, without coming up with a new name. Unless that is what is called by infrastructure, in which case possibly consistency is valuable.

@oskarth what do you think?